cura/UI/PrintJobOutputModel.py

```diff
@@ -1,16 +1,15 @@
 # Copyright (c) 2018 Ultimaker B.V.
 # Cura is released under the terms of the LGPLv3 or higher.
 
-from PyQt5.QtCore import pyqtSignal, pyqtProperty, QObject, pyqtSlot
 from typing import Optional, TYPE_CHECKING, List
 
-from PyQt5.QtCore import QUrl
+from PyQt5.QtCore import pyqtSignal, pyqtProperty, QObject, pyqtSlot, QUrl
 from PyQt5.QtGui import QImage
 
 if TYPE_CHECKING:
     from cura.PrinterOutput.PrinterOutputController import PrinterOutputController
-    from cura.PrinterOutput.PrinterOutputModel import PrinterOutputModel
-    from cura.PrinterOutput.ConfigurationModel import ConfigurationModel
+    from cura.UI.PrinterOutputModel import PrinterOutputModel
```

Comment:
I'm not really a big fan of moving everything into UI. It does indeed influence the UI, but UI is just too generic as a format. The original location of the model is a better indicator of what it does and with what it interacts.

Reply:
I'll move them, again. From the discussion, I guess we agree on a structure as below:
```
cura.Machine  - machine related code
cura.Machine.Models - Qt models related to code in cura.Machine
cura.PrinterOutput
cura.PrinterOutput.Models
```
I'll change `cura.UI` to `cura.GUI` and move the misc models used in the GUI into `cura.GUI.Models` if you agree.